extensions/wikia/ContributionPrototype/CPArticleRenderer.php

```diff
@@ -45,17 +45,17 @@ public function __construct($publicHost, $wikiId, $dbName, $urlProvider) {
 	 * @param string $title
 	 * @param OutputPage $output
 	 */
-	public function render($title, OutputPage $output) {
-		$content = $this->getArticleContent($title);
+	public function render($title, OutputPage $output, $action='view') {
+		$content = $this->getArticleContent($title, $action);
 		
 		if ($content === false) {
 			// TODO: what do we want to show here?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)